lib/activerecord_batch_update.rb

```diff
@@ -45,15 +45,15 @@ def batch_update_statements(entries, update_on: :id, batch_size: 100)
       entries.map(&:stringify_keys).group_by { _1.keys.sort! }.sort.flat_map do |(keys, items)|
         next [] if keys.empty?
 
-        where_clause = where_statement(update_on)
-        update_clause = update_statement(keys - update_on)
+        where_clause = batch_update_where_statement(update_on)
+        update_clause = batch_update_statement(keys - update_on)
 
         items.each_slice(batch_size).map do |slice|
           [
-            "WITH \"#{cte_table.name}\" (#{keys.join(', ')})",
-            "AS ( #{values_statement(slice, keys)} )",
+            "WITH \"#{batch_update_table.name}\" (#{keys.join(', ')})",
+            "AS ( #{batch_update_values_statement(slice, keys)} )",
             update_clause,
-            "FROM \"#{cte_table.name}\"",
+            "FROM \"#{batch_update_table.name}\"",
             "WHERE #{where_clause}"
           ].join(' ')
         end
```

Comment:
we're waiting for some of [these PRs](https://github.com/rails/rails/pulls/benedikt) to be merged to get rid of string interpolation here ⌛